CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+## THIS SHOULD ALWAYS BE LAST ##
+* @BitGo/btc-team
```

Comment:
I think you misunderstood my feedback. The catch-all codeowner needs to be first so this comment needs to be updated to reflect that. There should still be another entry for `.github` and `renovate.json` where infra shares the ownership with the corresponding team

Reply:
my bad. have pushed one more time.